src/Behat/Behat/EventDispatcher/ServiceContainer/EventDispatcherExtension.php

```diff
@@ -153,4 +154,23 @@ protected function loadEventDispatchingStepTester(ContainerBuilder $container)
         $definition->addTag(TesterExtension::STEP_TESTER_WRAPPER_TAG, array('priority' => -9999));
         $container->setDefinition(TesterExtension::STEP_TESTER_WRAPPER_TAG . '.event_dispatching', $definition);
     }
+
+    /**
+     * Loads ticking step tester.
+     *
+     * @param ContainerBuilder $container
+     */
+    protected function loadTickingStepTester(ContainerBuilder $container)
+    {
+        if (!function_exists('pcntl_signal')) {
+            return;
+        }
+
+        $definition = new Definition('Behat\Behat\EventDispatcher\Tester\TickingStepTester', array(
+            new Reference(TesterExtension::STEP_TESTER_ID),
+            new Reference(self::DISPATCHER_ID)
```

Comment:
the second parameters looks useless as you don't even acknowledge it in the `TickingStepTester` class ? (there is only one argument in there)